modules/roles_profiles/manifests/profiles/gecko_t_osx_1014_generic_worker.pp

```diff
@@ -6,11 +6,18 @@
 
     require roles_profiles::profiles::cltbld_user
 
+    $worker_type = 'gecko-t-osx-1014'
+
     case $::operatingsystem {
         'Darwin': {
 
             include puppet::atboot
 
+            class { 'fluentd':
```

Comment:
Move this to a generic 'logging' profile and then include it on the gecko 1014 role.